styles/header.css

```diff
@@ -0,0 +1,98 @@
+.main-screen {
+  background-image: url(../img/index-background-desktop.svg);
+  background-repeat: no-repeat;
+  background-position: right;
+  padding-top: 54px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__menu {
+  display: flex;
+  justify-content: space-between;
+  max-width: 477px;
+  width: 100%;
+}
+
+.header__logo-link:hover {
+  opacity: 0.6;
+}
+
+.header__logo-link:active {
+  opacity: 0.3;
+}
+
+.header__link {
+  font-size: 20px;
+  color: #ffffff;
+}
+
+.header__link:hover {
+  opacity: 0.6;
+}
+
+.header__link:active {
+  opacity: 0.3;
+}
+
+.header__link--active {
+  position: relative;
+}
+
+.header__link--active::after {
+  content: "";
+  position: absolute;
+  top: 35px;
+  left: 0;
+  width: 100%;
+  height: 2px;
+  background-color: #ffffff;
+}
+
+.main-screen__content {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  position: relative;
+  left: 83px;
+  bottom: 24px;
+}
+
+.main-screen__title {
+  font-size: 60px;
+  margin-bottom: 33px;
+  max-width: 450px;
+  line-height: 60px;
+  text-transform: none;
+}
+
+.main-screen__description {
+  font-size: 20px;
+  margin-bottom: 53px;
+}
+
+.main-screen__button {
+  width: 245px;
+  height: 52px;
+  font-size: 20px;
+  color: #ffffff;
+  background-color: #68b738;
+}
+
+.main-screen__button:hover {
+  color: #ffffff;
+  background-color: #5eaa2f;
+}
+
+.main-screen__button:active {
+  background-color: #5eaa2f;
+  color: rgba(255, 255, 255, 0.302);
```

Comment:
2 digits are often enough after dot. `0.3` is fine in this case.

Reply:
fixed